integration_tests/sdk/aqueduct_tests/delete_workflow_test.py

```diff
@@ -79,7 +79,9 @@ def test_force_delete_workflow_saved_objects(
 ):
     """Check the flow with object(s) saved with update_mode=APPEND can only be deleted if in force mode."""
     table_name = generate_table_name()
-    table_artifact = data_integration.sql(query=SHORT_SENTIMENT_SQL_QUERY)
+    table_artifact = data_integration.sql(
+        "select * from %s limit 5" % format_table_name("hotel_reviews", data_integration.type())
+    )
```

Comment:
How come this is only used on this one aqueduct test? Don't we do this many times in `aqueduct_test/`?

Reply:
Changed it to just the query without formatting concerns